new_course/1_programming_fundementals/methods.md

```diff
@@ -176,29 +176,62 @@ end
 puts my_name #=> "Joe Smith"
 ~~~
 
+This can be useful for example in a situation where you want your method to check for possible errors before continuing:
+
+~~~ruby
+def even_odd(number)
+  unless number.is_a? Numeric
+    return "A number was not entered"
+  end
+
+  if number % 2 == 0
+    "That is an even number."
+  else
+    "That is an odd number."
+  end
+end
+
+puts even_odd(20) #=>  That is an even number.
+puts even_odd("Ruby") #=>  A number was not entered.
+~~~
+
+Try removing the explicit `return` from the method above. Did the method run as expected?
+
 #### Difference Between `puts` and `return`
 
 A common source of confusion for new programmers is the difference between `puts` and `return`.
 
- * `puts` is a method that prints whatever argument you pass it to the console.
- * `return` is the final output of a method that you can use in other places throughout your code. 
+ * `puts` is a method that simply prints a given value to the console.
+ * `return` produces a value that can be used in your code. 
```

Comment:
I prefer the original versions here since they're a bit more explicit in their explanations.